Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1545,6 +1554,14 @@ void CClientVehicle::SetHeliRotorSpeed(float fSpeed)
     m_fHeliRotorSpeed = fSpeed;
 }
 
+void CClientVehicle::SetPlaneRotorSpeed(float fSpeed)
+{
+    if (m_pVehicle && m_eVehicleType == CLIENTVEHICLE_PLANE)
+        m_pVehicle->SetPlaneRotorSpeed(fSpeed);
```

Comment:
You also have to apply `m_fPlaneRotorSpeed` like we do with `m_fHeliRotorSpeed` when the vehicle gets created.

https://github.com/multitheftauto/mtasa-blue/blob/e66cc7a4623f65d46a4b711371197c2ba57aa926/Client/mods/deathmatch/logic/CClientVehicle.cpp#L2551

Reply:
Indeed, without that method call chances are that the plane rotor speed won't apply properly on stream-in. Good catch!

Reply:
@gta191977649 Can you do this?

Reply:
ah yeah, nice catch

Reply:
> @gta191977649 Can you do this?

Yeah, sorry for the long wait :x